easybuild/easyblocks/m/molpro.py

```diff
@@ -67,85 +88,95 @@ def configure_step(self):
-        else:
-            raise EasyBuildError("Failed to determine LAUNCHER value")
-
-        # determine full installation prefix
-        prefix_regex = re.compile('^PREFIX=(.*)$', re.M)
-        res = prefix_regex.search(cfgtxt)
-        if res:
-            self.full_prefix = res.group(1)
-            self.log.debug("Found full installation prefix: %s", self.full_prefix)
-        else:
-            raise EasyBuildError("Failed to determine full installation prefix")
+            # determine original LAUNCHER value
+            launcher_regex = re.compile('^LAUNCHER=(.*)$', re.M)
+            res = launcher_regex.search(cfgtxt)
+            if res:
+                self.orig_launcher = res.group(1)
+                self.log.debug("Found original value for LAUNCHER: %s", self.orig_launcher)
+            else:
+                raise EasyBuildError("Failed to determine LAUNCHER value")
+
+            # determine full installation prefix
+            prefix_regex = re.compile('^PREFIX=(.*)$', re.M)
+            res = prefix_regex.search(cfgtxt)
+            if res:
+                self.full_prefix = res.group(1)
+                self.log.debug("Found full installation prefix: %s", self.full_prefix)
+            else:
+                raise EasyBuildError("Failed to determine full installation prefix")
 
-        # determine MPI launcher command that can be used during build/test
-        # obtain command with specific number of cores (required by mpi_cmd_for), then replace that number with '%n'
-        launcher = self.toolchain.mpi_cmd_for('%x', self.cfg['parallel'])
-        launcher = launcher.replace(' %s' % self.cfg['parallel'], ' %n')
+            # determine MPI launcher command that can be used during build/test
+            # obtain command with specific number of cores (required by mpi_cmd_for), then replace that number with '%n'
+            launcher = self.toolchain.mpi_cmd_for('%x', self.cfg['parallel'])
+            launcher = launcher.replace(' %s' % self.cfg['parallel'], ' %n')
 
-        # patch CONFIG file to change LAUNCHER definition, in order to avoid having to start mpd
-        for line in fileinput.input(cfgfile, inplace=1, backup='.orig'):
-            line = re.sub(r"^(LAUNCHER\s*=\s*).*$", r"\1 %s" % launcher, line)
-            sys.stdout.write(line)
+            # patch CONFIG file to change LAUNCHER definition, in order to avoid having to start mpd
+            for line in fileinput.input(cfgfile, inplace=1, backup='.orig'):
+                line = re.sub(r"^(LAUNCHER\s*=\s*).*$", r"\1 %s" % launcher, line)
+                sys.stdout.write(line)
 
-        # reread CONFIG and log contents
-        cfgtxt = read_file(cfgfile)
-        self.log.info("Contents of CONFIG file:\n%s", cfgtxt)
+            # reread CONFIG and log contents
+            cfgtxt = read_file(cfgfile)
+            self.log.info("Contents of CONFIG file:\n%s", cfgtxt)
+
+    def build_step(self):
```

Comment:
@boegel: You suggested build_step probably isn't necessary on account of the superclass method being chosen by default. How does that work given that we only want to run build_step if a compilation needs to be done?